stacks/container-apps/cms-alarms.yml

```diff
@@ -0,0 +1,115 @@
+# stacks/container-apps/cms-alarms.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: CMS alarms
+Conditions:
+  CreateProductionResources: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  OpsFatalMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AudioSqsCallbackName:
+    Type: AWS::SSM::Parameter::Value<String>
+  ImageSqsCallbackName:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  # callback queues are being slow
+  CmsAudioCallbackSlowAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[CMS][Worker][SQS] ${EnvironmentType} Audio Callbacks Slow"
+      AlarmActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      InsufficientDataActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      OKActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      AlarmDescription: Audio callbacks are behind
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 3
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Average
+      Threshold: 60
```

Comment:
Okay, updated: so if there's any message > 1 minute old for 3 consecutive minutes ... error alarm.